nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/serde/FlatBuffersMapper.java

```diff
@@ -741,13 +754,31 @@ public static int asFlatNode(@NonNull SameDiff sameDiff, @NonNull DifferentialFu
                     dtypeArgs[e] = (byte) d[e].toInt();
                 }
             }
+
+            if(dynamicCustomOp.numSArguments() > 0) {
+                sArgs = new String[dynamicCustomOp.numSArguments()];
+                extraStringIds = new int[dynamicCustomOp.numSArguments()];
+                val sArgs2 = dynamicCustomOp.sArgs();
+                for(int i = 0; i < sArgs2.length; i++) {
+                    sArgs[i] = sArgs2[i];
```

Comment:
this looks a bit suspect.  Why can't we assign `sArgs = dynamicCustomOp.sArgs()`?

Reply:
You're right. This is a copy anyways. Updated.